src/main/starlark/builtins_bzl/common/cc/cc_helper.bzl

```diff
@@ -1236,6 +1272,7 @@ cc_helper = struct(
     get_local_defines_for_runfiles_lookup = _get_local_defines_for_runfiles_lookup,
     are_labels_equal = _are_labels_equal,
     get_srcs = _get_srcs,
+    get_module_interfaces = _get_module_interfaces,
```

Comment:
nit: may I suggest calling this `get_cpp20module_interfaces`. That way the code related to cpp20modules will be easier to locate. I'd reserve the name without `cpp20` only for new attrs on the rules.